### DISCUSSION TITLE
Standardize outputs for DELLY and Manta?

### DISCUSSION THREAD

**Discussion by WuSelina** [uclahs-cds/pipeline-call-sSV#97] [Q&A] [answered]
Hello! 
I am wondering if it is possible to have the outputs from DELLY and Manta match, as in both be in binary (BCF) or not (VCF). Right now, it looks like DELLY outputs are in binary. 

I am also wondering if the information outputted by Manta's four compressed VCFs is contained in the one DELLY BCF output. 
To confirm, do the Manta outputs only contain somatic SVs? 
Thank you so much for your help.

**Comment by Faizal-Eeman** [comment, marked as answer]
Thanks Selina,
Issue #98 created to compress Manta VCF outputs to BCF.

> I am also wondering if the information outputted by Manta's four compressed VCFs is contained in the one DELLY BCF output. To confirm, do the Manta outputs only contain somatic SVs?

For somatic SV calling, you'd consider `somaticSV.vcf.gz` from the Manta output as SVs and indels are scored under a somatic variant model. See this [Manta doc](https://github.com/Illumina/manta/blob/master/docs/userGuide/README.md#structural-variant-predictions) for description of each VCF output. The other VCFs appear to have unfiltered variant calls. So, variants called in `somaticSV.vcf.gz` from Manta may also be called in the DELLY BCF output.

**Reply by WuSelina** [reply]
Thanks Faizal! Do you think it's better to output both Manta and DELLY outputs as BCFs or VCFs?

**Reply by Faizal-Eeman** [reply]
BCFs would be better as they are compressed, so lesser file size.
